io/io/inc/ROOT/RFile.hxx

```diff
@@ -23,13 +24,17 @@ class TKey;
 namespace ROOT {
 namespace Experimental {
 
+class RKeyInfo;
 class RFile;
-struct RFileKeyInfo;
 
 namespace Internal {
 
 ROOT::RLogChannel &RFileLog();
 
+/// Returns an **owning** pointer to the object referenced by `key`. The caller must delete this pointer.
+/// This method is meant to only be used by the pythonization.
+[[nodiscard]] void *GetRFileObjectFromKey(RFile &file, const RKeyInfo &key);
```

Comment:
I think I would prefer a special handling of `Get<void>(...)` that doesn't check types.

Reply:
I mean in `RField::Get` in general, which would then make this function not needed.